src/docs/software/using/python.md

```diff
@@ -0,0 +1,116 @@
+Python is an interpreted, object-oriented, high-level programming language with dynamic semantics.
+Its high-level built in data structures, combined with dynamic typing and dynamic binding, 
+make it very attractive for Rapid Application Development, as well as for use as a scripting or 
+glue language to connect existing components together. Python's simple, easy to learn syntax 
+emphasizes readability and therefore reduces the cost of program maintenance. 
+
+
+## Python versions
+Sherlock features two different versions of Python: 2.7 and 3.6.1. 
+The reason for the two versions is that some code, or modules, will only work with old features 
+of version 2 while most code with work on version 3. 
+The general consensus is to use Python 3 whenever you can, especially for most numerical based modules 
```

Comment:
I agree in principle, but there's a variety of Python 2.7-only modules on Sherlock, and that tends to be the default version when both are available.

Reply:
Also, the things that are installed via `pip` are technically called `packages`, not `modules`. I guess that's also easier for users to distinguish between things they load with `module load` (modules) and things they install with (`pip`) packages.